cvlr-log/src/log.rs

```diff
@@ -88,7 +88,7 @@ impl<T: CvlrLog> CvlrLog for &T {
 impl CvlrLog for &str {
     #[inline(always)]  
     fn log(&self, _tag: &str, logger: &mut CvlrLogger) {
-        logger.log(*self);
+        logger.log(self);
```

Comment:
Are you sure this is correct and deref will be added automatically? If so, this is good